### PR TITLE
[REFACTORING] Refactored admin navigation

### PR DESCRIPTION
**Pull request by christianbader**

### CODE REVIEW

File: Api/Shipping.php [hunk 1]
```diff
@@ -595,7 +597,9 @@ public function setOrder(\Sulu\Bundle\Sales\OrderBundle\Entity\Order $order = nu
      */
     public function getOrder()
     {
-        return new Order($this->entity->getOrder(), $this->locale);
+        $productFactory = new ProductFactory();
+        $itemFactory = new ItemFactory($productFactory, 'EUR');
+        return new Order($this->entity->getOrder(), $this->locale, $itemFactory);
```
Comment (notsure) [inline Api/Shipping.php]: new line before return

File: Api/ShippingItem.php [hunk 2]
```diff
@@ -134,7 +136,8 @@ public function setItem(\Sulu\Bundle\Sales\CoreBundle\Entity\Item $item)
      */
     public function getItem()
     {
-        return new Item($this->entity->getItem(), $this->locale);
+        $productFactory = new ProductFactory();
+        return new Item($this->entity->getItem(), $this->locale, $productFactory);
```
Comment (notsure) [inline Api/ShippingItem.php]: add newline before return

File: Api/ShippingItem.php [hunk 3]
```diff
@@ -24,7 +25,8 @@ class ShippingItem extends ApiWrapper
      * @param ShippingItemEntity $entity
```
Comment (notsure) [inline Api/ShippingItem.php:27]: check all doc types in this file

File: Api/Shipping.php [hunk 4]
```diff
@@ -522,10 +563,10 @@ public function getItems()
     /**
      * Set status
      *
-     * @param \Sulu\Bundle\Sales\ShippingBundle\Entity\ShippingStatus $status
+     * @param ShippingStatus $status
      * @return Shipping
```
Comment (ghost) [inline Api/Shipping.php:567]: Blank line before return

File: Controller/NavigationController.php [hunk 5]
```diff
@@ -1,40 +0,0 @@
...
-namespace Sulu\Bundle\Sales\ShippingBundle\Controller;
-
-use Sulu\Bundle\AdminBundle\Admin\ContentNavigation;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Symfony\Component\HttpFoundation\Response;
-
-
-/**
- * Class NavigationController
- * @package Sulu\Bundle\Sales\ShippingBundle\Controller
```
Comment (xandrej) [inline Controller/NavigationController.php]: Blank line

**Comment by ghost** [comment]
ACK

**Pull request merged**